google-account-plugin/src/com/google/cloud/tools/intellij/login/CredentialedUser.java

```diff
@@ -111,20 +106,18 @@ protected void setActive(boolean isActive) {
   }
 
   private void initializeUserInfo(Userinfoplus userInfo,
-      final IGoogleLoginCompletedCallback updateUserCallback) {
+      @Nullable final IGoogleLoginCompletedCallback callback) {
     if (userInfo == null) {
       name = null;
       image = null;
     } else {
       name = userInfo.getName();
-      IUserPropertyCallback pictureCallback = new IUserPropertyCallback<Image>() {
-        @Override
-        public void setProperty(Image newImage) {
-          image = newImage;
-          updateUserCallback.onLoginCompleted();
+      GoogleLoginUtils.provideUserPicture(userInfo, newImage -> {
+        image = newImage;
+        if (callback != null) {
+          callback.onLoginCompleted();
```

Comment:
ugh. my suggested name will make this line ugly, but I think that's the fault of the interface definition. I'm fine with `loginCompletedCallback` or something like that to make this less ugly.